csrc/quantization/cutlass/scaled_mm_dq_sm8x.cu

```diff
@@ -0,0 +1,217 @@
+#include <torch/extension.h>
+
+#include <assert.h>
+#include <stddef.h>
+
+// clang-format will break include orders
+// clang-format off
+#include "cute/tensor.hpp"
+#include "cute/atom/mma_atom.hpp"
+#include "cutlass/numeric_types.h"
+
+#include "cutlass/util/device_memory.h"
+
+#include "cutlass/cutlass.h"
+#include "cutlass/gemm_coord.h"
+#include "cutlass/arch/arch.h"
+#include "cutlass/arch/mma.h"
+#include "cutlass/gemm/device/gemm.h"
+#include "cutlass/gemm/device/gemm_universal_adapter.h"
+
+#include "cutlass/epilogue/threadblock/fusion/visitors.hpp"
+#include "cutlass/gemm/kernel/default_gemm_universal_with_visitor.h"
+
+#include "cutlass_visitor_2x_broadcast_epilogue.hpp"
+#include "common.hpp"
+// clang-format on
+
+/////////////////////////////////////////
+
+template <typename Arch, typename ElementAB_, typename ElementD_>
```

Comment:
Can you add a comment that describes the operation in human readable notation?  I think even a cut+paste of the equivalent python code would be helpful.